includes/sanitizers/class-amp-iframe-sanitizer.php

```diff
@@ -135,8 +135,17 @@ public function sanitize() {
 				if ( $node->parentNode->parentNode instanceof DOMElement && 'figure' === $node->parentNode->parentNode->tagName ) {
 					$figure_node = $node->parentNode->parentNode;
 				}
-				if ( $figure_node && $figure_node->hasAttribute( 'class' ) && in_array( 'alignfull', explode( ' ', $figure_node->getAttribute( 'class' ) ), true ) ) {
-					$normalized_attributes['layout'] = 'responsive';
+
+				if ( $figure_node && $figure_node->hasAttribute( 'class' ) ) {
+
+					$figure_node_classes = explode( ' ', $figure_node->getAttribute( 'class' ) );
+
+					// Changes layout to responsive for videos which have wide or full width (contains alignwide or alignfull class).
+					$show_responsive = ( in_array( 'alignfull', $figure_node_classes, true ) || in_array( 'alignwide', $figure_node_classes, true ) );
+
+					if ( true === $show_responsive ) {
+						$normalized_attributes['layout'] = 'responsive';
+					}
```

Comment:
Minor formatting suggestion:

```suggestion
					$figure_node_classes = explode( ' ', $figure_node->getAttribute( 'class' ) );

					// If the alignment was set to 'wide width' or 'full width',  set the layout to responsive.
					$show_responsive = (
						in_array( 'alignfull', $figure_node_classes, true ) ||
						in_array( 'alignwide', $figure_node_classes, true )
					);

					if ( true === $show_responsive ) {
						$normalized_attributes['layout'] = 'responsive';
					}
```

Reply:
Thanks, I have formatted the code as per the suggestion.